front/src/modules/auth/queries/update.ts

```diff
@@ -85,3 +87,45 @@ export const RENEW_TOKEN = gql`
     }
   }
 `;
+
+// TODO: Fragments should be used instead of duplicating the user fields !
```

Comment:
Good point, let's make the effort in the next PRs and introduce some examples for everyone.